app/views/sections/_title.html.erb

```diff
@@ -0,0 +1,85 @@
+<%# TODO
+  refactor this into a `Profile` model
+  that knows how to display a person,
+  whether they have a response plan or not.
+%>
```

Comment:
This could also be taken care of by creating separate actions for:

* `drafts#show`
* `approvals#show`
* `repsonse_plans#show`

Also, I'm thinking that the controllers might benefit from a renaming:
`approvals` -> `submissions`, `response_plans` -> `published_plans`.